docs/03/00.md

```diff
@@ -90,6 +90,18 @@ project_url [https://github.com/n8han/my-proj]:
 developer_url [https://github.com/n8han]:
 ```
 
+### Conditionals
```

Comment:
Could you please make a more elaborate example? It would be nice if it included `else` or `else if` (if it's supported). It could be some real use-case example, like depending on a version of some library using different snippet of code.